sunpy/map/tests/test_mapbase.py

```diff
@@ -1500,7 +1499,6 @@ def test_contour_units(simple_map):
         assert np.all(c1 == c2)
 
 
-@pytest.mark.skipif(Version(matplotlib.__version__) < Version("3.6.0"), reason="Fails on old MPL versions, the first with block raises a different error")
 def test_contour_inputs(simple_map):
     with pytest.raises(ValueError, match='Contour levels must be increasing'):
         simple_map.draw_contours([10, -10] * u.dimensionless_unscaled)
```

Comment:
Having non-increasing contour levels apparently no longer emits an error, so this block can be deleted.

Reply:
We should add a test to see what non-increasing contour levels does to the output.